tasks.py

```diff
@@ -121,6 +121,39 @@ def init_database(ctx, batch_end_epoch=None, mins_ago=None, override_empty=False
     cursor.close()
     conn.close()
 
+@task
+def create_ro_user(ctx):
+    db_host = os.environ.get("POSTGRES_HOST")
+    db_port = os.environ.get("POSTGRES_PORT")
+    db_name = os.environ.get("POSTGRES_DB")
+    db_user = os.environ.get("POSTGRES_USER")
+    db_password = os.environ.get("POSTGRES_PASSWORD")
+    db_ro_user = os.environ.get("POSTGRES_RO_USER")
+    db_ro_password = os.environ.get("POSTGRES_RO_PASSWORD")
```

Comment:
Perhaps it would be good to include this in the readme (https://github.com/MinaFoundation/uptime-service-validation?tab=readme-ov-file#postgres-configuration) saying that we have this invoke task also and that there are those two new env variables, etc.

Reply:
Good point! Updated the readme.